spotbugs/src/main/java/edu/umd/cs/findbugs/AbstractBugReporter.java

```diff
@@ -49,9 +51,7 @@
  * BugReporter objects.
  */
 public abstract class AbstractBugReporter implements BugReporter {
-    private static final boolean DEBUG = SystemProperties.getBoolean("abreporter.debug");
-
-    private static final boolean DEBUG_MISSING_CLASSES = SystemProperties.getBoolean("findbugs.debug.missingclasses");
+    private static final Logger LOG = LoggerFactory.getLogger(AbstractBugReporter.class);
```

Comment:
To make copy-pasting easier, you can use `LoggerFactory.getLogger(MethodHandles.lookup().lookupClass())`